src/fixtures/ad.html

```diff
@@ -17,6 +17,8 @@
         align-items: center;
       }
     </style>
+    <meta name="amp-cta-type" content="LEARN_MORE">
+    <meta name="amp-cta-url" content="https://www.amp.dev">
```

Comment:
Nit: No `www.` ;)

Reply:
All of the templates had it that way :open_mouth:

Reply:
ah, aight, no problem then.